synapse/streams/__init__.py

```diff
@@ -32,4 +32,4 @@ async def get_new_events(
         is_guest: bool,
         explicit_room_id: Optional[str] = None,
     ) -> Tuple[List[R], K]:
-        ...
+        raise NotImplementedError()
```

Comment:
I wonder if this should be an `abstractmethod`?

Reply:
And this is 29dd442461.